File: src/main/java/com/adven/concordion/extensions/exam/db/DbResultRenderer.java
package com.adven.concordion.extensions.exam.db;

import com.adven.concordion.extensions.exam.rest.RestResultRenderer;
import org.concordion.api.listener.AssertEqualsListener;
import org.concordion.api.listener.AssertFalseListener;
import org.concordion.api.listener.AssertSuccessEvent;
import org.concordion.api.listener.AssertTrueListener;

public class DbResultRenderer extends RestResultRenderer
        implements AssertEqualsListener, AssertTrueListener, AssertFalseListener {

    @Override
    public void successReported(AssertSuccessEvent event) {
        event.getElement().addStyleClass("table-success").appendNonBreakingSpaceIfBlank();
    }
}

File: src/main/java/com/adven/concordion/extensions/exam/db/DbResultRenderer.kt
package com.adven.concordion.extensions.exam.db

import com.adven.concordion.extensions.exam.ws.RestResultRenderer
import org.concordion.api.listener.AssertEqualsListener
import org.concordion.api.listener.AssertFalseListener
import org.concordion.api.listener.AssertSuccessEvent
import org.concordion.api.listener.AssertTrueListener

class DbResultRenderer : RestResultRenderer(), AssertEqualsListener, AssertTrueListener, AssertFalseListener {
    override fun successReported(event: AssertSuccessEvent) {
        event.element.addStyleClass("table-success").appendNonBreakingSpaceIfBlank()
    }
}